src/catalog/catalog.cpp

```diff
@@ -98,6 +99,21 @@ std::vector<TableSchema*> Catalog::getTableSchemas(
     return result;
 }
 
+std::string Catalog::getNodeTableCypher(
```

Comment:
I would move this function to physical operator instead. Feels that no other component will call this. In fact are we still using this function?